File: Python/Program/Human.py
# Saya Mia Karisma Haq NIM [2102165] mengerjakan soal Latihan Praktikum-2 dalam mata kuliah DPBO 
# untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin 

# Kelas Human yang digunakan untuk merepresentasikan data Human, Human merupakan kelas induk dari kelas SivitasAkademik dan buyut dari kelas Mahasiswa
class Human:
    # Atribut yang digunakan pada kelas Human terdiri dari nik, nama, dan jenis kelamin. Karena pada dasarnya setiap manusia dibedakan berdasarkan 3 identitas tersebut.
    __nik = ""
    __nama = ""
    __jenis_kelamin = ""

    # Constructor
    def __init__(self, nik="", nama="", jenis_kelamin=""):
        self.__nik = nik
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin

    # Method setter untuk set nilai setiap atribut pada kelas Human
    def setNik(self, nik):
        self.__nik = nik

    def setNama(self, nama):
        self.__nama = nama
    
    def setJenisKelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin

    # Method getter untuk mendapatkan nilai setiap atribut pada kelas Human
    def getNik(self):
        return self.__nik

    def getNama(self):
        return self.__nama

    def getJenisKelamin(self):
        return self.__jenis_kelamin

File: PHP/Program/Human.php
<!-- 
    Saya Mia Karisma Haq NIM [2102165] mengerjakan soal Latihan Praktikum-2 dalam mata kuliah DPBO
    untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin 
-->

<!-- 
    Kelas Human yang digunakan untuk merepresentasikan data Human, Human merupakan kelas induk dari kelas SivitasAkademik dan buyut dari kelas Mahasiswa 
-->
<?php
    class Human
    {
        // Atribut yang digunakan pada kelas Human terdiri dari nik, nama, dan jenis kelamin. Karena pada dasarnya setiap manusia dibedakan berdasarkan 3 identitas tersebut.
        private $nik = '';
        private $nama = '';
        private $jenis_kelamin = '';
        
        // Constructor
        public function __construct($nik = '', $nama = '', $jenis_kelamin = '')
        {
            $this->nik = $nik;
            $this->nama = $nama;
            $this->jenis_kelamin = $jenis_kelamin;
        }

        // Method setter untuk set nilai setiap atribut pada kelas Human
        public function setNik($nik)
        {
            $this->nik = $nik;
        }

        public function setNama($nama)
        {
            $this->nama = $nama;
        }

        public function setJenisKelamin($jenis_kelamin)
        {
            $this->jenis_kelamin = $jenis_kelamin;
        }

        // Method getter untuk mendapatkan nilai setiap atribut pada kelas Human
        public function getNik()
        {
            return $this->nik;
        }

        public function getNama()
        {
            return $this->nama;
        }

        public function getJenisKelamin()
        {
            return $this->jenis_kelamin;
        }

        // Destructor
        function __destruct()
        {
            
        }
    }
?>